### PR TITLE
Enable multithreaded build on Travis CI

### PR DESCRIPTION
**Pull request by antongovorov**
Added changes in .travis.yml to build in parallel mode, namely added -j2 parameter on "make install"

Related: [APPLINK-28056](https://adc.luxoft.com/jira/browse/APPLINK-28056)

### CODE REVIEW

**Comment by antongovorov** [comment]
@AByzhynar @anosach-luxoft @AGaliuzov please review

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ before_install:
 script:
   - sudo ln -sf /usr/bin/gcov-4.9 /usr/bin/gcov
   - bash -e tools/infrastructure/check_style.sh
-  - mkdir build && cd build && cmake ../ -DBUILD_TESTS=ON -DENABLE_GCOV=ON && make install
+  - mkdir build && cd build && cmake ../ -DBUILD_TESTS=ON -DENABLE_GCOV=ON && make install -j2
```
Comment (pestOO) [inline .travis.yml]: @antongovorov 
Maybe `-j$(nproc)`?

Comment (antongovorov) [inline .travis.yml]: @pestOO 
There are two proc on build machine, as i know . When i build on my local machine with same parameters, all ok.

Comment (antongovorov) [inline .travis.yml]: @pestOO 
What mean -j$(nproc). Where i could read about this command?

Comment (pestOO) [inline .travis.yml]: @antongovorov 
http://bfy.tw/7o3H

**Pull request closed without merging**